config/crawl.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const Crawler = require("simplecrawler");
+const chalk = require('chalk');
+
+const app = express();
+
+app.use(express.static(`${__dirname}/../_site`));
```

Comment:
Maybe check that `_site` has contents, otherwise the call to `refs.every(...)` later on (and potentially other things) will fail.


Here's output from running `yarn crawl` without first running `yarn build`:

```sh
/Users/jamesseppi/CODE/web-design-standards-docs/config/crawl.js:67
        const isWarning = refs.every(path => WARNING_PAGES.includes(path));
                              ^

TypeError: Cannot read property 'every' of undefined
    at notFound.forEach.item (/Users/jamesseppi/CODE/web-design-standards-docs/config/crawl.js:67:31)
```